docs/architecture/porting-existing-aspnet-apps/additional-migration-resources.md

```diff
@@ -0,0 +1,55 @@
+author: ardalis
+ms.date: 11/13/2020
+---
+
+# Additional migration resources
+
+As you're planning and executing your migration from ASP.NET MVC and/or Web API to ASP.NET Core, there are a number of resources available to help beyond this book. Make a note of these and leverage them where appropriate to help you overcome obstacles you encounter on your migration journey.
+
+## Official documentation
+
+The official documentation website, [docs.microsoft.com](https://docs.microsoft.com/), has the most up-to-date information available about versions, frameworks, breaking changes, and support options. You'll find many links in this book to docs articles, but for any problem you're facing it's often worth at least doing a quick search of the docs to see if there is already information there covering the issue and offering a solution or workaround.
+
+## Github
+
+Because .NET Core is an open source project, many issues are discovered, reported, discussed, and fixed on GitHub. Microsoft has several GitHub organizations in which you'll find repositories that may be helpful. A partial list of these organizations and some of their public repositories are listed below:
+
+- [Microsoft](https://github.com/microsoft)
+  - [ASP.NET API Versioning](https://github.com/microsoft/aspnet-api-versioning)
+- [dotnet](https://github.com/dotnet)
+  - [ASP.NET Core](https://github.com/dotnet/aspnetcore)
+  - [.NET Runtime](https://github.com/dotnet/runtime)
+  - [Entity Framework Core](https://github.com/dotnet/efcore)
+  - [C# Language](https://github.com/dotnet/csharplang)
+  - [Docs](https://github.com/dotnet/docs)
+  - [Docs Samples](https://github.com/dotnet/samples)
+  - [Try Convert](https://github.com/dotnet/try-convert)
+- [.NET Architecture Reference Apps](https://github.com/dotnet-architecture)
+  - [eShopModernizing](https://github.com/dotnet-architecture/eShopModernizing)
+  - [eShopOnWeb](https://github.com/dotnet-architecture/eShopOnWeb)
+  - [eShopOnContainers](https://github.com/dotnet-architecture/eShopOnContainers)
+
+## Stack Overflow
+
+Stack Overflow has a wealth of information in the form of previous questions asked and answers given, with the most helpful answers listed first and marked if they solved the problem. In addition to searching for an existing solution to a problem you may encounter, you can of course also ask a question yourself and hope for some response from the .NET community. Don't forget you can narrow down a search by using tags, and remember to use appropriate tags when you ask questions to maximize the chances of someone with the experienced needed noticing your question.
+
+## YouTube channels
+
+YouTube has a huge amount of .NET and .NET Core video content, which may include useful tutorials or walkthroughs covering any scenario you may encounter. Consider searching it separately if your other efforts to find help online come up short.
+
+## Twitter, Gitter, Slack, etc.
```

Comment:
```suggestion
## Twitter, Gitter, Slack, and other community channels
```

Per style guide, avoid Latin abbreviations for common English phrases - `etc.` caught my eye.